src/components/Offramp/Confirm.view.tsx

```diff
@@ -301,43 +310,58 @@ export const OfframpConfirmView = ({
         address: string,
         claimLinkData: any, // TODO: fix type
         chainId: string,
-        tokenAddress: string
-    ) => {
+        tokenAddress: string,
+        isSameChain?: boolean // e.g. for opt ETH -> opt USDC
```

Comment:
No - this is because of PR 457 https://github.com/peanutprotocol/peanut-ui/pull/457/files

when same chain but still token swap, the destination chain hash is same as source chain hash, and has to be handled differently